src/api/python/_api.py

```
@@ -705,6 +705,58 @@ def release_xfer_handle(self, handle: nixl_xfer_handle):
+
+    @overload
+    def prep_mem_view(
+        self,
+        mem_type: str,
+        descs: list[tuple],
+        *,
+        backends: list[str] = [],
+    ) -> int:
+        ...
+
+    def prep_mem_view(self, *args, backends: list[str] = []) -> int:
```

Ruff (0.15.15)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)